lib/msal-browser/src/config/Configuration.ts

```diff
@@ -38,6 +39,7 @@ export type BrowserAuthOptions = {
     navigateToLoginRequestUrl?: boolean;
     clientCapabilities?: Array<string>;
     protocolMode?: ProtocolMode;
+    refreshTokenBinding?: boolean;
```

Comment:
We may consider putting this under `system`.

Reply:
Done.